pkg/controller/registry/reconciler/grpc.go

```diff
@@ -192,7 +192,7 @@ func (c *GrpcRegistryReconciler) EnsureRegistryServer(catalogSource *v1alpha1.Ca
 	source := grpcCatalogSourceDecorator{catalogSource}
 
 	// if service status is nil, we force create every object to ensure they're created the first time
```

Comment:
nit: should we update this comment?

```suggestion
	// if service status is nil, or it is invalid, we force create every object to ensure they're created the first time
```